tools/ansible/tasks/wsu/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- hosts: localhost
+  vars:
+    project_root: "{{ playbook_dir }}/../../../.."
+    kubelet_location: "https://dl.k8s.io/v1.14.0/kubernetes-node-windows-amd64.tar.gz"
+    tmp_path: "/tmp/wsu"
+
+  tasks:
+    - name: Gather required files
+      block:
+
+        - name: Create directory to store files to be used later
+          file:
+            path: "{{ tmp_path }}"
+            state: directory
+
+        - name: Build WMCB
+          make:
+            target: build
+            chdir: "{{ project_root }}"
+        - name: Move WMCB to temporary directory
+          command: mv "{{ project_root }}/wmcb.exe" "{{ tmp_path }}/wmcb.exe"
```

Comment:
Can't we register the variable in `Build WMCB` instead of specifying `wmcd.exe`?

Reply:
Good point, will change